utils/scarb-ui/src/args/features.rs

```diff
@@ -0,0 +1,18 @@
+use clap::Parser;
+use smol_str::SmolStr;
+
+/// Features
+#[derive(Parser, Clone, Debug)]
+pub struct FeaturesSpec {
+    /// Comma separated list of features to activate.
+    #[arg(short = 'F', long, value_delimiter = ',', env = "SCARB_FEATURES")]
+    pub features: Vec<SmolStr>,
```

Comment:
Yes, exactly. 
`SmolStr` is just small optimization that let's you avoid heap allocation for some strings. We use want to use it for some values in the package manifest, as we might create many. It does not make sense on cli level though and we want to limit the number of deps in `scarb-ui` as this is a crate published to crates.io, that can potentially be used by other apps. We use it in FeatureOps on Scarb side though, to avoid conversions later on when we compare it with manifests contents.